docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -681,32 +809,133 @@ Suppose the output of `stats.workload.value` looked something like this:
 
-These rough calculations give you a lower bound to the required throughput, which is _at least_ 440 tasks per minute to ensure recurring tasks are executed, at their scheduled time. This throughput doesn't account for nonrecurring tasks that might have been scheduled, nor does it account for tasks (recurring or otherwise) that might be scheduled in the future.
+These rough calculations give you a lower bound to the required throughput, which is _at least_ 410 tasks per minute to ensure recurring tasks are executed, at their scheduled time. This throughput doesn't account for nonrecurring tasks that might have been scheduled, nor does it account for tasks (recurring or otherwise) that might be scheduled in the future.
 
 Given these inferred attributes, it would be safe to assume that a single {kib} instance with default settings **would not** provide the required throughput. It is possible that scaling horizontally by adding a couple more {kib} instances will.
 
 For details on scaling Task Manager, see <<task-manager-scaling-guidance>>.
 
+
+[[task-manager-health-evaluate-the-capacity-estimation]]
+===== Evaluate the Capacity Estimation
+
+Task Manager is constantly evaluating its runtime operations and workload. This enables Task Manager to make rough estimates about the sufficiency of its capacity.
+
+As the name suggests, these are estimates based on historical data and should not be used as predictions. These estimations should be evaluated along side the detailed <<task-manager-health-monitoring>> stats before making changes to infrastructure. These estimations assumes all {kib} instances are configured identically.
+
+We recommend using these estimations when following the Task Manager <<task-manager-scaling-guidance>>.
+
+Evaluating the health stats above you can see the following output under `stats.capacity_estimation.value`:
+
+[source,json]
+--------------------------------------------------
+{
+  "observed": {
+    "observed_kibana_instances": 1, # <1>
+    "minutes_to_drain_overdue": 1, # <2>
+    "max_throughput_per_minute_per_kibana": 200,
+    "max_throughput_per_minute": 200, # <3>
+    "avg_recurring_required_throughput_per_minute": 28, # <4>
+    "avg_recurring_required_throughput_per_minute_per_kibana": 28,
+    "avg_required_throughput_per_minute": 28, # <5>
+    "avg_required_throughput_per_minute_per_kibana": 28
+  },
+  "proposed": {
+    "min_required_kibana": 1, # <6>
+    "proposed_kibana": 1, # <7>
+    "avg_recurring_required_throughput_per_minute_per_kibana": 28,
+    "avg_required_throughput_per_minute_per_kibana": 28
+  }
+}
+--------------------------------------------------
+<1> These estimates assume that there is one {kib} instance actively executing tasks
```

Comment:
To me `observed_kibana_instances` means the number of Kibanas running, while this description says `estimate`. Is this an estimate or an actual number?

Reply:
It's an estimate based on the observed number of OwnerIds.
There is no way, currently, for Kibana to know how many instances are running, which is frustrating but we have to work with it.
In the code you'll see a bunch of calculations where we use the term "assumed" - as in, we assume this top be the case, but it's not 100% as we're basing this off of the observed number of Kibana. In order for us to produce any kind of estimate for the non-recurring we need some kind of assumption to work off of, and so observing the OwnerIds is the closest we can come to producing an estimate.

Reply:
I spoke to @chrisronline and he thinks Stack Monitoring could provide us a more reliable number.
I'll file a follow up issue for us to look into that in the near future, so we can improve this incrementally.